components/pipeline-service/development/main-pipeline-service-configuration.yaml

```diff
@@ -1960,6 +1960,136 @@ spec:
                       requests:
                         cpu: 200m
                         memory: 200Mi
+        pipelines-as-code-controller:
+          spec:
+            template:
+              spec:
+                containers:
+                  - name: pac-controller
+                    resources:
+                      limits:
+                        cpu: 500m
+                        memory: 500Mi
+                      requests:
+                        cpu: 500m
+                        memory: 500Mi
+        pipelines-as-code-watcher:
+          spec:
+            template:
+              spec:
+                containers:
+                  - name: pac-watcher
+                    resources:
```

Comment:
I think those are way too high for the development overlay, I'll suggest cpu: 500, memory: 1 or 2 GB. Basically for the development overlay I think we should use values as found on the staging clusters which usually do not see any load. The thing here is to pass the CI tests, without being too greedy for resources.
This applies for all development overlay values.